lib/rules/alphabetize-properties.js

```diff
@@ -49,26 +59,34 @@ module.exports = {
       throw new Error("Value of `exempt` option must be an array of strings (key names); saw a: " + typeof exemptions);
     }
     var removeExemptions = makeExemptionsFilter(exemptions);
+    function checkExport(node) {
+      if (!node.specifiers.length) {
+        return;
+      }
+      var rawExportNames = node.specifiers.map(extractName);
+      if (rawExportNames.length < limit) {
+        return;
+      }
+      var exportNames = rawExportNames.filter(removeExemptions);
+      if (exportNames.find(checkOrderOfKeysIn(exportNames))) {
+        context.report({message: "Exports with "+limit+" or more specifiers should have specifiers in alphabetical order. (Saw "+rawExportNames.length+")", node: node});
```

Comment:
Could we create the message using template strings?

Reply:
I had been keeping this as plain-JS as possible, with the hopes that it'll be able to work on as older versions of JS/ES... But that might not be as not as important as the readability that template strings and arrow functions offer?

Reply:
Ohh good point, then this is fine! :)